### PR TITLE
Winamp layer fixes

### PR DESCRIPTION
**Pull request by myst6re**

### CODE REVIEW

File: src/audio.cpp [hunk 1]
```diff
@@ -215,6 +215,11 @@ void NxAudioEngine::setMusicSpeed(float speed)
 	_engine.setRelativePlaySpeed(_musicHandle, speed);
 }
 
+void NxAudioEngine::setMusicLooping(bool looping)
+{
+	_engine.setLooping(_musicHandle, false);
```
Comment (julianxhokaxhiu) [inline src/audio.cpp:220]: Was this supposed to be `_engine.setLooping(_musicHandle, looping);`?

File: src/audio.h [hunk 2]
```diff
@@ -73,6 +73,7 @@ class NxAudioEngine
 	float getMusicVolume();
 	void setMusicVolume(float _volume, size_t time = 0);
 	void setMusicSpeed(float speed);
+	void setMusicLooping(bool looping);
```
Comment (julianxhokaxhiu) [inline src/audio.h:76]: Maybe we can set this parameter to `false` by default, in order to match your previous line :)

**Review by julianxhokaxhiu — CHANGES_REQUESTED** [review]

File: src/audio/winamp/winamp.cpp [hunk 3]
```diff
@@ -73,14 +73,32 @@ namespace SoLoud
...
+	{
+		// Can resume music
+		if (aInfoKey == 7777) {
+			// Needs to Duplicate
+			return 1.0f;
+		}
+
+		return 0.0f;
+	}
+
```
Comment (julianxhokaxhiu) [inline src/audio/winamp/winamp.cpp:101]: This section needs to be added to VGMStream too I guess, as the whole flow of "can I resume" is called in `music.cpp` therefore, also VGMStream is impacted by this change. Would you be able to add the required logic over there too?

**Review by julianxhokaxhiu — CHANGES_REQUESTED** [review]

File: src/audio/winamp/winamp.cpp (hunk 3 above)
Comment (myst6re) [reply src/audio/winamp/winamp.cpp:101]: Yes I can do that
Just know that vgmstream is currently not impacted. getInfo returns 0.0 in AudioSourceInstance parent implementation.

Comment (julianxhokaxhiu) [reply src/audio/winamp/winamp.cpp:101]: Ah ok so if there's no impact then it's not needed :) Thanks! Fine by you if I merge this then?

Comment (myst6re) [reply src/audio/winamp/winamp.cpp:101]: No don't merge it, I have more changes to push :)

**Comment by myst6re** [comment]
TODO: edit features in README, change ps2_bios_path description in configuration file.

**Pull request closed without merging**